backend/src/mailer/text/transactionReceived.ts

```diff
@@ -25,6 +26,9 @@ ${data.memo}
 Bitte antworte nicht auf diese E-Mail!
 
 Mit freundlichen Grüßen,
-dein Gradido-Team`,
+dein Gradido-Team
+
+
+Link zu deiner Übersicht: ${data.overviewURL}`,
```

Comment:
```suggestion
Link zu deinem Konto: ${data.overviewURL}`,
```

Reply:
As far I can see the changes in `deployment/bare_metal/.env.dist` are there:

- https://github.com/gradido/gradido/blob/942e39534b84dc420c7ee3ff5c9fdbe80368bfcb/deployment/bare_metal/.env.dist#L21
- https://github.com/gradido/gradido/blob/942e39534b84dc420c7ee3ff5c9fdbe80368bfcb/deployment/bare_metal/.env.dist#L47